src/api/public/apidocs-new/paths/attribute_namespace_meta.yaml

```diff
@@ -0,0 +1,174 @@
+get:
+  summary: Shows attribute namespace meta data.
+  description: Shows attribute namespace meta data.
```

Comment:
What is meta data?

Reply:
This was clarified privately. This description got replaced by "Shows attribute namespace".